uni/ulsp/database.icn

```diff
@@ -143,70 +167,72 @@ class LSPDB(
                file_without_ext := tab(upto("."))
             }
 
-            package_db[pack.getName()]["files"][file_without_ext] := table()
-            if /package_db[pack.getName()]["files"][file_without_ext]["imports"] then package_db[pack.getName()]["files"][file_without_ext]["imports"] := []
+
+            file_table[file_without_ext] := table()
+
+            if /file_table[file_without_ext]["imports"] then file_table[file_without_ext]["imports"] := import_table := []
```

Comment:
Add a line break after `then`, i.e:
```
if /file_table[file_without_ext]["imports"] then
   file_table[file_without_ext]["imports"] := import_table := []
```
And since you are doing an if null test, you don't really do use `if` at all, can be just written as 
```
/file_table[file_without_ext]["imports"] := (import_table := [])
```